include/pmacc/nvidia/atomic.hpp

```diff
@@ -23,12 +23,18 @@
 
 
 #include "pmacc/types.hpp"
-#if( PMACC_CUDA_ENABLED == 1 )
+#include "pmacc/memory/Array.hpp"
+#if( BOOST_LANG_CUDA || BOOST_COMP_HIP)
 #   include "pmacc/nvidia/warp.hpp"
 #endif
 #include <boost/type_traits.hpp>
 #include <climits>
 
+#include <alpaka/intrinsic/Traits.hpp>
+#include <alpaka/warp/Traits.hpp>
+
```

Comment:
Is the new line separation of the includes (line 32 and line 35) intended?

Reply:
I think it's now a little weird that standard library includes are separated by alpaka ones.